classes/utils/Biome.py

```diff
@@ -0,0 +1,95 @@
+from enum import Enum
+
+
+class Biome(Enum):
+
+    def __init__(self, id, string):
+        self.id = id
+        self.string = string
+
+    def id_to_biome(id):
+        for biome in Biome:
+            if biome.id == id:
+                return biome
+        return None
+
+    def name_to_biome(name):
+        for biome in Biome:
+            if biome.name == name:
+                return biome
+        return None
+
+    def string_to_biome(string):
+        for biome in Biome:
+            if biome.biome == string:
+                return biome
+        return None
+
+    INVALID_BIOME = (-1, 'Invalid Biome')
+    OCEAN = (0, 'Ocean')
+    PLAINS = (1, 'Plain')
+    DESERT = (2, 'Desert')
+    MONTAINS = (3, 'Montains')
+    EXTREME_HILLS = (3, 'ExtremeHills')
+    FOREST = (4, 'Forest')
+    TAIGA = (5, 'Taiga')
+    SWAMPLAND = (6, 'Swampland')
+    RIVER = (7, 'River')
+    HELL = (8, 'Hell')
+    NETHER = (8, 'Nether')
+    SKY = (9, 'Sky')
```

Comment:
This biome doesn't exist

Reply:
True, I've took the enum from another minecraft project (cuberite), I'll edit these

Reply:
You should autogen it https://wiki.vg/Data_Generators